scripts/drone/steps/lib.star

```diff
@@ -406,12 +406,12 @@ def build_frontend_step(edition, ver_mode):
     # TODO: Use percentage for num jobs
     if ver_mode == 'release':
         cmds = [
-            './bin/grabpl build-frontend --jobs 8 ' + \
+            './bin/build build-frontend --jobs 8 ' + \
```

Comment:
I'm likely missing something but why do we have this go code which eventually runs `yarn build`? Can't we just run that here instead? 

```suggestion
            'yarn build'
```

Reply:
So two reasons:
1. Because we want to be able to specify how many jobs can run concurrently until the FE is built.
2. Because currently it does more than that, see: https://github.com/grafana/grafana/pull/53073/files#diff-c50d7d5858920a5d2533d6c0c2c5d32de3b459ad291498e9c44e8345a0ad9fe2

If you think that this is no longer needed - be simplified, let us know and I guess me and Kevin will be more than happy to reduce complexity!

Reply:
It would also be fun to see what will happen if we don't provide the `--jobs` flag there.
I know that @wbrowne had some bad experience here, but it may not be the case for our Drone runners?

Reply:
I think that'd be a good candidate for a follow-up PR. This was just intended to be a no-op from what we already have to this.

The fact that you were able to identify an opportunity to make it more concise is exactly the reason we're doing this. :)